index.js

```diff
@@ -4,7 +4,7 @@ var required = require('requires-port')
   , lolcation = require('./lolcation')
   , qs = require('querystringify')
   , relativere = /^\/(?!\/)/
-  , protocolre = /^([a-z0-9.+-]+:)?(\/\/)?(.*)$/i; // actual protocol is first match
+  , protocolre = /^([a-z0-9.+-]+:)?(\/\/)?([\S\s]*)$/i; // actual protocol is first match
```

Comment:
Now that you are at it I would sligtly improve this by:

1) removing the final `$` (I think it's really useless now) and we will save 1 byte :smile:.
2) use `[a-zA-Z][a-zA-Z0-9.+-]+:` instead of `[a-z0-9.+-]+:` to follow RFC [specifications](https://tools.ietf.org/html/rfc3986#section-3.1).